stream-transformation-tool-service/src/main/java/uk/gov/justice/tools/eventsourcing/transformation/service/EventStreamTransformationService.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.justice.tools.eventsourcing.transformation.service;
+
+import static java.util.function.Function.identity;
+
+import uk.gov.justice.services.eventsourcing.source.core.EventSource;
+import uk.gov.justice.services.eventsourcing.source.core.EventStream;
+import uk.gov.justice.services.eventsourcing.source.core.exception.EventStreamException;
+import uk.gov.justice.services.messaging.JsonEnvelope;
+import uk.gov.justice.tools.eventsourcing.transformation.api.EventTransformation;
+
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Stream;
+
+import javax.inject.Inject;
+
+/**
+ * Service to transform events contained on an event-stream.
+ */
+public class EventStreamTransformationService {
+
+    @Inject
+    EventSource eventSource;
+
+    Set<EventTransformation> transformations;
```

Comment:
Is populating this another task?

Reply:
Yea, this needs to be made part of the Swarm app, with the sideloaded jar.

But we don't have that to integrate yet so I just made it work for the sake of testing.

Martin and I will do that integration next week when we turn this into the Swarm App.